src/ocm/components/clusterConfiguration/DiscoveryImageTypeDropdown.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import {
+  DropdownItem,
+  DropdownToggle,
+  Dropdown,
+  FormGroup,
+  DropdownSeparator,
+} from '@patternfly/react-core';
+import { CaretDownIcon } from '@patternfly/react-icons';
+import { useField } from 'formik';
+import { getFieldId } from '../../../common';
+
+const discoveryImageTypes = {
+  'minimal-iso': 'Minimal image file - Provision with virtual media',
+  'full-iso': 'Full image file - Provision with physicial media',
+  'discovery-image-ipxe': 'iPXE - Provision from your network server',
+};
+
+type DiscoveryImageTypeDropdownProps = {
+  name: string;
+  defaultValue: string | undefined;
+  onChange: (imageType: string) => void;
+};
+
+export const DiscoveryImageTypeDropdown = ({
+  name,
+  defaultValue,
+  onChange,
+}: DiscoveryImageTypeDropdownProps) => {
+  const [field, , { setValue }] = useField(name);
+  const [isOpen, setOpen] = React.useState(false);
+  const [current, setCurrent] = React.useState(defaultValue);
+  const fieldId = getFieldId(name, 'input');
+  const dropdownItems = [
+    <DropdownItem
+      key="minimal-iso"
+      id="minimal-iso"
+      description={
+        'Use when your storage capacity is limited or being served over a constrained network.'
+      }
+    >
+      {discoveryImageTypes['minimal-iso']}
+    </DropdownItem>,
+    <DropdownSeparator key="separator1" />,
+    <DropdownItem
+      key="full-iso"
+      id="full-iso"
+      description={'The generated discovery ISO will contain everything needed to boot.'}
+    >
+      {discoveryImageTypes['full-iso']}
+    </DropdownItem>,
+    <DropdownSeparator key="separator2" />,
+    <DropdownItem
+      key="discovery-image-ipxe"
+      id="discovery-image-ipxe"
+      description={'Use when you have an iPXE server that has already been set up.'}
+    >
+      {discoveryImageTypes['discovery-image-ipxe']}
+    </DropdownItem>,
+  ];
+
+  const onSelect = React.useCallback(
+    (event?: React.SyntheticEvent<HTMLDivElement>) => {
+      const currentValue = event?.currentTarget.id
+        ? (discoveryImageTypes[event?.currentTarget.id] as string)
+        : defaultValue;
+      setCurrent(currentValue ? currentValue : '');
+      setValue(event?.currentTarget.id);
+      setOpen(false);
+      onChange(event?.currentTarget.id || '');
+    },
+    [setCurrent, setOpen, defaultValue, setValue, onChange],
+  );
+
+  const toggle = React.useMemo(
+    () => (
+      <DropdownToggle
+        onToggle={(val) => setOpen(val)}
+        toggleIndicator={CaretDownIcon}
+        isText
+        className="pf-u-w-100"
+      >
+        {current}
+      </DropdownToggle>
+    ),
+    [setOpen, current],
+  );
+
+  return (
+    <FormGroup fieldId={fieldId} label={'Provisioning type'}>
+      <Dropdown
```

Comment:
Looks like the dropdown still doesn't remember the selected image-type.
![image-type](https://user-images.githubusercontent.com/829045/207372540-96594909-45fc-43e4-82a6-c4500b7c00a3.png)